### PR TITLE
Add sub flow while processing jinja2 computed attribute

### PR DESCRIPTION
**Pull request by dgarros**
In order to have a dedicated flow per node when we are updating a Jinja2 computed attribute, I've added a sub-flow `update_computed_attribute_value_jinja2` into the mix

Right now the flow is executed on the same worker (similar to what we are currently doing) so I didn't have to create a deployment for it
To be consistent it might be better to have a deployment and to use `submit_workflow` for it as well

### CODE REVIEW

File: backend/infrahub/computed_attribute/tasks.py [hunk 1]
```diff
@@ -158,6 +160,57 @@ async def trigger_update_python_computed_attributes(
         )
 
 
+@flow(
+    name="process_computed_attribute_value_jinja2",
+    flow_run_name="Update value for computed attribute {attribute_name}",
+)
+async def update_computed_attribute_value_jinja2(
+    branch_name: str, obj: CoreNode, attribute_name: str, template_value: str
```
Comment (ogenstad) [inline backend/infrahub/computed_attribute/tasks.py:168]: I think we could use something more descriptive than "obj" here.

**Review by ogenstad — APPROVED** [review]
>To be consistent it might be better to have a deployment and to use submit_workflow for it as well

Did we have any restrictions with regards to the types of objects we could send here? Is it anything that's JSON serialisable?

File: backend/infrahub/computed_attribute/tasks.py (hunk 1 above)
Comment (dgarros) [reply backend/infrahub/computed_attribute/tasks.py:168]: Initially I used `node` but it's currently a reserved keyword for the batch so it wouldn't work

**Comment by dgarros** [comment]
> Did we have any restrictions with regards to the types of objects we could send here? Is it anything that's JSON serialisable?

No restriction, thinking more about it we should probably just use a deployment here to be consistent
I'll do that in another PR

**Pull request merged**